compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -607,15 +607,32 @@ internal actual class ComposeWindow : UIViewController {
 
             override fun pointInside(point: CValue<CGPoint>, event: UIEvent?): Boolean =
                 point.useContents {
-                    val hitsInteropView = attachedComposeContext?.scene?.mainOwner?.hitInteropView(
-                        pointerPosition = Offset(
-                            (x * density.density).toFloat(),
-                            (y * density.density).toFloat()
-                        ),
-                        isTouchEvent = true,
-                    ) ?: false
-
-                    !hitsInteropView
+                    val position = Offset(
+                        (x * density.density).toFloat(),
+                        (y * density.density).toFloat()
+                    )
+
+                    // TODO: temporary solution copying control flow from [ComposeScene.processPress]
```

Comment:
FYI agreed that it's temporary - it shouldn't replicate this logic (that will be refactored soon). More correct way to send touches to scene as black box and handle only that ones that were received in interop view (compose layout inside owner)

Reply:
Yeah, agree. This is a rather trivial solution and solves a reported issue, so I don't see harm in it.